apm-sniffer/apm-agent-core/src/main/java/org/skywalking/apm/agent/core/plugin/interceptor/enhance/InstanceMethodsAroundInterceptor.java

```diff
@@ -12,31 +12,33 @@
     /**
      * called before target method invocation.
      *
-     * @param context            instance context, a class instance only has one {@link EnhancedClassInstanceContext} instance.
+     * @param context instance context, a class instance only has one {@link EnhancedClassInstanceContext} instance.
      * @param interceptorContext method context, includes class name, method name, etc.
-     * @param result             change this result, if you want to truncate the method.
+     * @param result change this result, if you want to truncate the method.
+     * @throws Throwable
      */
     void beforeMethod(EnhancedClassInstanceContext context, InstanceMethodInvokeContext interceptorContext,
-                      MethodInterceptResult result);
+        MethodInterceptResult result) throws Throwable;
```

Comment:
All plugins did't care how to process the exception, that is the framework of agent's duty.